### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by matikups** [futurepress/epubjs-rn#133]
I'd like to know if this project is still maintained, since there are no reply's from the author in issues.

I must say, I really like this library. 
But I cannot use it on Android properly.

**Comment by fchasen** [comment]
Yep still maintained, but have fallen a bit behind with releases.
Will review the PRs early next week.

**Comment by cllemon** [comment]
I am also the same, encountered some problems on Android, thanks to the author's continued maintenance.

**Comment by matikups** [comment]
Thanks @fchasen @cllemon for your kind replies.
@fchasen this is the issue: https://github.com/futurepress/epubjs-rn/issues/129

I've commented everything in there, it's something about a polyfill, though I tried to add the package of babel manually with no luck. It's something about Object.assign.
I'll stand by (:

**Comment by matikups** [comment]
I'd like to get it up and running, since I would like very much to contribute to this project.
I could run in iOS flawlessly, I'm trying to make a better swipe mode 👍
